src/extra/Markdown.tsx

```diff
@@ -0,0 +1,736 @@
+import memoize = require('lodash/memoize');
+import * as marked from 'marked';
+import * as React from 'react';
+import { MarkdownProps } from 'rendition/dist/extra/Markdown';
+import { Txt } from '../';
+
+const markedOptions = {
+	// Enable github flavored markdown
+	gfm: true,
+	headerIds: false,
+	sanitize: true,
+};
+
+const render = memoize((text: string = '') => marked(text, markedOptions));
+
+import styled from 'styled-components';
+
+/*
+ Taken From: https://github.com/sindresorhus/github-markdown-css
+ */
+const GitHubMarkdown = styled(Txt)`
```

Comment:
May I suggest to have a different file just for this huge constant? Spanning 707 line, I think that it deserves having its own place.

Reply:
I disagree, I think moving the style to another file would make the code less readable, not more readable.

Reply:
I just realized how slim is the actual component. Alternatively, how about combining the two into something like:
```ts
export const Markdown = styled(({ children, ...props }: MarkdownProps) => {
	return (
		<Txt
			{...props}
			dangerouslySetInnerHTML={{ __html: render(children) }}
		/>
	);
})`
...CSS Rules here...
`;
```
like we do in the UI. No strong opinion on this though.